python/sdk/merlin/batch/sink.py

```diff
@@ -18,6 +18,7 @@
 
 import client
 from merlin.batch.big_query_util import valid_table_id, valid_column
+from merlin.batch.maxcompute_util import mc_valid_table_id, mc_valid_columns
```

Comment:
ultra nit: can we rename `valid_table_id` to `bq_valid_table_id` or something? Same for `bq_valid_column`?

Reply:
Fixed